src/Tools/dotnet-openapi/test/OpenApiAddTests.cs

```diff
@@ -0,0 +1,245 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.IO;
+using System.Text.RegularExpressions;
+using System.Threading;
+using System.Threading.Tasks;
+using System.Xml;
+using Microsoft.DotNet.OpenApi.Tests;
+using Microsoft.DotNet.Tools;
+using Microsoft.Extensions.Internal;
+using Xunit;
+using Xunit.Abstractions;
+
+namespace Microsoft.DotNet.OpenApi.Add.Tests
+{
+    public class OpenApiAddTests : OpenApiTestBase
+    {
+        public OpenApiAddTests(ITestOutputHelper output) : base(output){}
+
+        [Fact]
+        public void OpenApi_Add_GlobbingOpenApi()
+        {
+            throw new NotImplementedException();
```

Comment:
Eh?

Reply:
I realized I hadn't accounted for this as I was sending the PR so I created a test that would fail every time to remind me to deal with it before making this a non-Design PR.